edgelet/provisioning/src/provisioning.rs

```diff
@@ -116,19 +126,53 @@ impl From<&str> for ReprovisioningStatus {
             "deviceDataReset" => ReprovisioningStatus::DeviceDataReset,
             "initialAssignment" => ReprovisioningStatus::InitialAssignment,
             _ => {
-                debug!("DPS provisioning result substatus {}", s);
+                debug!("Provisioning result substatus {}", s);
                 ReprovisioningStatus::InitialAssignment
             }
         }
     }
 }
 
+impl Display for ReprovisioningStatus {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{:?}", self)
```

Comment:
Why do you want to impl `Display` that forwards to `Debug` instead of using `Debug` directly?